src/coinbase/address.ts

```diff
@@ -259,7 +259,7 @@ export class Address {
    *
    * @param assetId - The ID of the asset to transfer from the faucet.
    * @returns The faucet transaction object.
-   * @throws {InternalError} If the request does not return a transaction hash.
+   * @throws {Error} If the request does not return a transaction hash.
    * @throws {Error} If the request fails.
```

Comment:
Can we collapse these into a singular `@throws` statement? `If the request fails` Not returning a transaction hash feels like an implementation detail here.

Can follow-up on this